### ISSUE TITLE
Issue on split screen view, popup view and landscape view

### ISSUE THREAD

**Issue by merbin2012** [EMI-INDO/emi-indo-cordova-plugin-admob#47]
1) In the "split screen" view and popup view two problems
a) Might need to not take the system navbar height.
b) While the banner ad is load in split view, it show the following error (tried with "leaderboard" ad type)
         `Error building request URL: Ad size will not fit on screen. a_w=728,  a_h=90,  s_w=384,  s_h=374`

2) In landscape mode, no need to take the navbar height

A temporary solution is to disable those modes, by adding the following line on the AndroidManifest.xml file

```
<activity
    android:screenOrientation="portrait" 
    android:resizeableActivity="false" <!-- Disables split-screen and popup view -->
</activity>
```

**Comment by EMI-INDO** [comment]
I don't want this change to bother other users, because it affects (UX), so maybe tomorrow I add a variable to cordova plugin.xml and capacitor, the default value remains true. 

- I found some new issues in android 15/16, and problems when using plugins cordova-plugin-statusbar or 
cordova-plugin-fullscreen
- if you have time you can test the above 2 problems

**Comment by merbin2012** [comment]
Is the temporary fix working or not? I have not checked it on Android 15 yet, but I will check it later.

**Comment by EMI-INDO** [comment]
I have tested the statusBar still cannot be displayed on android 15/16

the point is how can the plugin display statusBar on android 15/16 so as not to damage `emiAdmobPlugin.styleBannerAd`

- I want to make an option how in android 15/16 the statusBar remains displayed.
- like the only way to use hooks is to disable edge-to-edge node:tools = “remove”

**Comment by merbin2012** [comment]
I think it doesn't affect the capacitor project. Please confirm by the following image.
![Capacitor](https://github.com/user-attachments/assets/f9426ab6-8f27-46ff-8ae5-b576d584c150)

Am I correct?

**Comment by EMI-INDO** [comment]
Yes unfortunately I don't have an Android 15/16 phone, I can't see what's happening with banner ads on real devices.

but in the user reviews of my app on play store it says the black empty space above the banner is quite big 😁 on their device android 15/16.

**Comment by merbin2012** [comment]
Are you adding a black line above the banner ad? If yes, can I see your JavaScript code?

I am doing like this

```
	let blackLine = document.getElementById("bannerBottomWhiteSpace");
	
	if (blackLine)
		blackLine.remove()

	blackLine = document.createElement("div");
	blackLine.id = "bannerBottomWhiteSpace";
	blackLine.style.width = "100%";
	blackLine.style.height = `${extraHeightForNavigationBar+parseInt(bannerHeightPx)+parseInt(lineHeightPx)}px`;
	blackLine.style.background = _bannerTopSpaceColor;
	blackLine.style.position = "fixed";//"relative";
	blackLine.style.bottom = 0;//`${bannerHeightPx}px`; // Position above the banner
	blackLine.style.zIndex = "99999"; // Ensure it's above the banner
	document.body.appendChild(blackLine);
```

**Comment by EMI-INDO** [comment]
https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/releases/tag/v2.0.0


I'm off for the next 5 to 7 days.
No updates.

Have a good day.
